test/unit/gateways/stripe_payment_intents_test.rb

```diff
@@ -26,12 +26,15 @@ def setup
 
     @apple_pay = apple_pay_payment_token
     @google_pay = network_tokenization_credit_card(
-      '4777777777777778',
-      payment_cryptogram: 'BwAQCFVQdwEAABNZI1B3EGLyGC8=',
-      verification_value: '987',
-      source: :google_pay,
+      '4242424242424242',
+      payment_cryptogram: 'reddelicious',
+      source: :android_pay,
```

Comment:
Why was this changed? Is it more accurate test data?
Are we sure there are no differences between the source being `android_pay` and `google_pay`?